service/src/test/java/bio/terra/javatemplate/iam/SamServiceTest.java

```diff
@@ -4,20 +4,21 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.Mockito.when;
 
+import bio.terra.javatemplate.BaseSpringBootTest;
 import org.broadinstitute.dsde.workbench.client.sam.ApiException;
 import org.broadinstitute.dsde.workbench.client.sam.api.StatusApi;
 import org.broadinstitute.dsde.workbench.client.sam.model.SystemStatus;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
 
 @ExtendWith(MockitoExtension.class)
-class SamServiceTest {
+class SamServiceTest extends BaseSpringBootTest {
 
-  @Mock private SamClient samClient;
-  @Mock private StatusApi statusApi;
+  @MockBean private SamClient samClient;
```

Comment:
I think the only place `MockBean` should be used is in a `WebMvcTest`. A service test should always use the service's constructor so it's clear that all dependent components are mocked and none are being injected "live" by accident.